uint64 frame
float64 angle
sensor_msgs/Image rotated_image 
